### PR TITLE
Validations and  Model Specs

### PR DESCRIPTION
**Pull request by Joel-100**
- For the User model,names are not blank.
- PostsCounter is an integer greater than or equal to zero.
- For the Post model,Title isn't blank and shouldn't exceed 250 characters.
- CommentsCounter is an integer greater than or equal to zero.
- LikesCounter is an integer greater than or equal to zero.
- Added unit specs for all models', methods and validations.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'test', posts_counter: 0) }
+
+  before { subject.save }
+
+  it 'should be valid' do
+    expect(subject).to be_valid
+  end
+
+  it 'should be valid' do
```
Comment (andersonlebon) [inline spec/models/user_spec.rb:12]: ## [OPTIONAL] 

- [ ] Kindly change this message to `should not be valid` as is testing the invalidity of the user.

File: spec/models/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'should be valid' do
```
Comment (andersonlebon) [inline spec/models/post_spec.rb]: ## [OPTIONAL] 

- [ ]  Kindly change this message to `should not be valid` as is testing the invalidity of the post.

File: spec/models/like_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```
Comment (andersonlebon) [inline spec/models/like_spec.rb]: ## [OPTIONAL] 
- [ ] Kindly correct the modal name it, should be `Like` and not `User`

**Review by andersonlebon — APPROVED** [review]
## Hi @Joel-100 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Joel-100) [reply spec/models/user_spec.rb:12]: Thanks!

File: spec/models/like_spec.rb (hunk 3 above)
Comment (Joel-100) [reply spec/models/like_spec.rb]: Ok thaks!

**Pull request merged**